avro-fastserde/src/main/java/com/linkedin/avro/fastserde/FastDatumReaderWriterUtil.java

```diff
@@ -55,6 +58,18 @@ public int hashCode() {
   private FastDatumReaderWriterUtil() {
   }
 
+  public static void warmUpFastGenericDatumReader(Schema writerSchema, Schema readerSchema, long timeoutInMillis)
```

Comment:
this is ~trivial enough (getFitire().get(timeout) that we can expect users to write this directly in their code i think ?

Reply:
thanks. I removed these warm up utils.